pkg/coinbase/staking_balance.go

```diff
@@ -95,7 +90,7 @@ func newStakingBalanceFromModel(m *client.StakingBalance) (*StakingBalance, erro
 
 	return &StakingBalance{
 		model:           m,
-		parsedDate:      date,
+		parsedDate:      m.GetDate(),
```

Comment:
staking balance date field changed from a string to a time.Time so no longer need this parsing.